pulsar-client-tools-test/src/test/java/org/apache/pulsar/client/cli/PulsarClientToolTest.java

```diff
@@ -183,13 +183,12 @@ public void testDurableSubscribe() throws Exception {
         String[] args = {"produce", "--messages", "Have a nice day", "-n", Integer.toString(numberOfMessages), "-r",
                 "20", "-p", "key1=value1", "-p", "key2=value2", "-k", "partition_key", topicName};
         Assert.assertEquals(pulsarClientToolProducer.run(args), 0);
-        Assert.assertFalse(future.isCompletedExceptionally());
-        future.get();
-        //wait for close
-        Awaitility.waitAtMost(Duration.ofDays(2000));
-        List<String> subscriptions = admin.topics().getSubscriptions(topicName);
-        Assert.assertNotNull(subscriptions);
-        Assert.assertEquals(subscriptions.size(), 1);
+
+        try {
+            future.get(10, TimeUnit.SECONDS);
+        } catch (Exception e) {
+            Assert.fail("consumer was unable to decrypt messages", e);
```

Comment:
"decrypt" -> "receive" since encryption isn't used in this test.

Reply:
Ok, I already fix it.